### ISSUE TITLE
Exclude some files

### ISSUE THREAD

**Issue by decentralizedb** [zehks/BitcoinFilesystemMonitoring#1]
I think could be better to exclude the following files from the statistics. Just in case someone has a lot of test data or a huge log file.

```.bitcoin
├── wallets
└── testnet3
      ├── blocks
      │   └── index
      ├── chainstate
      ├── indexes
      │   ├── blockfilter
      │   │   └── basic
      │   │       └── db
      │   └── txindex
      └── wallets
├── debug.log

**Comment by zehks** [comment]
Addressed on [d93f174](https://github.com/zehks/BitcoinFilesystemMonitoring/commit/d93f1748a1582722fcdaa950b50d2d118c7e68ed)

**Comment by decentralizedb** [comment]
Excellent, thanks, I'm using it!
